mysql-test/suite/innodb/t/doublewrite.test

```diff
@@ -48,130 +48,32 @@ insert into t1 values(4, repeat('-',12));
 insert into t1 values(5, repeat('.',12));
 commit work;
 
+flush table t1 for export;
+unlock tables;
+
 --echo # ---------------------------------------------------------------
 --echo # Test Begin: Test if recovery works if first page of user
 --echo # tablespace is full of zeroes.
 
-select space from information_schema.innodb_sys_tables
-where name = 'test/t1' into @space_id;
+select space into @space_id from information_schema.innodb_sys_tables
+where name = 'test/t1';
 
 begin;
 insert into t1 values (6, repeat('%', 12));
---echo # Ensure that dirty pages of table t1 are flushed.
-set global innodb_buf_flush_list_now = 1;
 
---source ../include/no_checkpoint_start.inc
+set global innodb_log_checkpoint_now = 1;
 
 --echo # Make the first page dirty for table t1
 set global innodb_saved_page_number_debug = 0;
 set global innodb_fil_make_page_dirty_debug = @space_id;
 
---echo # Ensure that dirty pages of table t1 are flushed.
-set global innodb_buf_flush_list_now = 1;
+SET GLOBAL innodb_max_dirty_pages_pct_lwm=0.0;
+SET GLOBAL innodb_max_dirty_pages_pct=0.0;
```

Comment:
Are both `innodb_log_checkpoint_now` and the additional buffer pool flushing via `innodb_max_dirty_page_pct` really needed?